### ISSUE TITLE
Excessive log messages in engine with hosting other than 'individually' with 2 claps on macOS

### ISSUE THREAD

**Issue by baconpaul** [free-audio/interop-tracker#21]
Hi!

1. Open BWS43b6 on macos latest
2. Put BWS in mode where hosting is 'by plugin' as opposed to 'individually' (or really any setting other than 'individually')
3. Pull in a clap and make a track
4. Make a new track and pull in a second instance of that same clap

When I run the engine under CLion I then see debug (red) messages in my console. When I do the above in production with BWS I see that filling up engine.log. It gets to half a meg in about 60 seconds.

```
Interval start: Something went wrong. code = 16
Interval finish: Something went wrong. code = 16
Interval start: Something went wrong. code = 22
Interval finish: Something went wrong. code = 16
Interval start: Something went wrong. code = 22
Interval finish: Something went wrong. code = 16
Interval start: Something went wrong. code = 22
Interval finish: Something went wrong. code = 16
Interval start: Something went wrong. code = 22
```

whether i do this with dexed, clap-saw-demo, hive, etc...

If I set to 'individually' there's no such problem.

**Comment by abique** [comment]
Should be fixed in next beta (4.3 beta 8).

**Comment by baconpaul** [comment]
Thanks!
